package_bag/tests.py

```diff
@@ -10,6 +10,7 @@
 
```

Comment:
Wondering if the `add_bags_to_db` and `copy_binaries` methods shouldn't be in the setUp method. Better yet, the `add_bags_to_db` method could be replaced by a [fixture](https://docs.djangoproject.com/en/3.2/topics/testing/tools/#fixture-loading).

You could either move the `test_discover_bags` to its own class OR I think you could probably update the `copy_binaries` method so it moves binaries to the right place for this test and doesn't do the unpacking and such. There appear to be a number of identical lines of code.